pkgdb/src/registry/floxpkgs/flake.nix.in

```diff
@@ -0,0 +1,270 @@
+# -*- mode: nix; -*-
+# ============================================================================ #
+#
+# Wraps `nixpkgs' to patch the package set for traversal by `pkgdb scrape'.
+#
+# !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+#
+# Packages should NEVER be consumed/built from this flake!
+# It exists to produce databases for `pkgdb'!
+#
+# !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+#
+#
+# ---------------------------------------------------------------------------- #
+
+{
+
+# ---------------------------------------------------------------------------- #
+
+  # `@NIXPKGS_URL@' is substituted at runtime.
+  inputs.nixpkgs.url = "@NIXPKGS_URL@";
+
+# ---------------------------------------------------------------------------- #
+
+  outputs = { nixpkgs, self, ... }: let
+
+# ---------------------------------------------------------------------------- #
+
+    # Nixpkgs Configuration
+    config = {
+      allowUnfree = true;
+      allowBroken = true;
+    }; # End `config'
+
+    # Modifications/rules to be applied.
+    #
+    # Example:
+    # {
+    #   allowRecursive    = [["legacyPackages" "x86_64-linux" "darwin"]];
+    #   disallowRecursive = [["legacyPackages" "x86_64-linux" "python3"]];
+    #   allowPackage      = [["legacyPackages" "x86_64-linux" "python3" "pip"]];
+    #   disallowPackage   = [["legacyPackages" "x86_64-linux" "gcc"]];
+    # }
+    #
+    # TODO: Aliases
+    rules = builtins.fromJSON ( builtins.readFile (self + "/rules.json") );
+
+
+# ---------------------------------------------------------------------------- #
+
+    lib = let
```

Comment:
suggestion (nb): i mean.. if we already assume that nixpkgs is nixpkgs, we could also just use `nixpkgs.lib`.
I do get the argument to have stable builtins based bundled here though..

Reply:
I think it's more prudent for our business logic to be separated from the nixpkgs rev that we're wrapping since that can change.